packages/languages/fa/src/index.ts

```diff
@@ -0,0 +1,15 @@
+import commonWords from './commonWords.json'
+import femaleFirstnames from './femaleFirstnames.json'
+import lastnames from './lastnames.json'
+import maleFirstnames from './maleFirstnames.json'
+import translations from './translations'
+
+export default {
+  dictionary: {
+    commonWords,
+    femaleFirstnames,
+    lastnames,
+    maleFirstnames,
```

Comment:
i need the sources for those dictionaries

Reply:
Sorry, I have no idea about what I done in past 3 hours.
There is NO WAY that I can find the source of these dictionaries.